CHANGES.txt

```diff
@@ -15,6 +15,13 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+ - onModule() method had been removed from io.crate.Plugin interface,
+   so createGuiceModules() must be used instead.
+
+ - ``srv`` and ``azure`` are no longer valid configuration options for ``discovery.type``.
```

Comment:
With some effort I think it would be possible to keep the compatibility here (ES doesn't have  this breaking change until 6.0) - but not sure if it's worth it